src/api/config/initializers/flipper.rb

```diff
@@ -0,0 +1,6 @@
+Flipper.configure do |config|
+  config.default do
+    adapter = Flipper::Adapters::ActiveRecord.new
+    Flipper.new(adapter)
```

Comment:
Hm is this really needed?

Reply:
Following [the documentation](https://github.com/jnunemaker/flipper/tree/master/docs/active_record#usage) on how to use the gem with the ActiveRecord adapter, yes.

Reply:
Tried it, it replaced the class with an instance in the main namespace...